docs/views/templates/check-your-answers.html

```diff
@@ -9,159 +9,184 @@
+            </ul>
           </dd>
         </div>
-
-        <div class="app-check-your-answers__contents">
-          <dt class="app-check-your-answers__question">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
             Home address
           </dt>
-          <dd class="app-check-your-answers__answer">
-            72 Guild Street<br />
-            London<br />
-            SE23 6FH
-          </dd>
-          <dd class="app-check-your-answers__change">
-            <a href="#">
-              Change<span class="govuk-visually-hidden"> home address</span>
-            </a>
+          <dd class="govuk-summary-list__value">
+            72 Guild Street
+            <br>London
+            <br>SE23 6FH
+          </dd>
+          <dd class="govuk-summary-list__actions">
+            <ul class="govuk-summary-list__actions-list">
+              <li class="govuk-summary-list__actions-list-item">
+                <a href="#">
+                  Change
+                  <span class="govuk-visually-hidden"> home address</span>
+                </a>
+              </li>
+            </ul>
           </dd>
         </div>
-
-        <div class="app-check-your-answers__contents">
-          <dt class="app-check-your-answers__question">
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key">
             Licence period
           </dt>
-          <dd class="app-check-your-answers__answer">
-            This is a longer paragraph of text provided by the user to provide additional information.<br /><br />
-            This is a second paragraph of text provided by the user.
-          </dd>
-          <dd class="app-check-your-answers__change">
-            <a href="#">
-              Change<span class="govuk-visually-hidden"> licence period</span>
-            </a>
+          <dd class="govuk-summary-list__value">
+            <p>This is a longer paragraph of text provided by the user to provide additional information.</p>
+            <p>This is a second paragraph of text provided by the user.</p>
+          </dd>
+          <dd class="govuk-summary-list__actions">
+            <ul class="govuk-summary-list__actions-list">
+              <li class="govuk-summary-list__actions-list-item">
+                <a href="#">
+                  Change
+                  <span class="govuk-visually-hidden"> licence period</span>
+                </a>
+              </li>
+            </ul>
           </dd>
         </div>
       </dl>
 
-
       <h2 class="govuk-heading-m">Now send your application</h2>
 
-      <p>
-        By submitting this notification you are confirming that, to the best of your knowledge, the details you are providing are correct.
-      </p>
+      <p>By submitting this notification you are confirming that, to the best of your knowledge, the details you are providing are correct.</p>
+
+      <form action="/url/of/next/page" method="post" novalidate>
```

Comment:
looks good, can we change this to `action="/confirmation"` for ease of prototyping?

Reply:
I've updated, only issue is that the `/` is probably needed when you copy this into the app folder, however having the `/` breaks the docs preview (which is current behaviour)

Reply:
thanks, as it's current behaviour I think that's ok